### ISSUE TITLE
New parser request: tsv

### ISSUE THREAD

**Issue by don-rumata** [kellyjonbrazil/jc#622]
Hi, can you add support for [tsv](https://en.wikipedia.org/wiki/Tab-separated_values)?

**Comment by kellyjonbrazil** [comment]
Yep, this should be an easy one. I can alias to the `csv` parser and just change the delimiter to `\t`.

**Comment by alexn-s** [comment]
tsv support would be great. is there an option or a flog to manually set a delimiter?
i could not find some.

by the way, i really like your tool

**Comment by bahuy110104** [comment]
maybe this will solve your problem [click](https://newrides.es/captha/)

**Comment by kellyjonbrazil** [comment]
There is no flag to set the delimiter but you could try using `tr`, `sed`, or similar to replace the existing delimiter to a comma.  Then run the output through the CSV parser.

**Comment by don-rumata** [comment]
Thanks!
